pkgs/development/python-modules/numdifftools/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage
+, lib
+, pytestpep8
+, pytest
+, pytestrunner
+, fetchPypi
+, numpy
+, scipy
+, statsmodels
+, algopy
+, hypothesis
+, line_profiler
+}:
+
+buildPythonPackage rec {
+  pname = "numdifftools";
+  version = "0.9.39";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1b1l8kpsw7qbxykcd0zl7gizvb36ij2fpxj4r937r6b56v2paz12";
+  };
+
+  checkInputs = [ hypothesis pytestpep8 pytest pytestrunner line_profiler ];
+  buildInputs = [ pytestrunner ];
```

Comment:
Is this really a required build input?

Reply:
Yeah, I tried to build it without pytestrunner, but it appears it is necessary...

Reply:
Could you add a comment with that info?